src/openapi.yaml

```diff
@@ -445,12 +445,6 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/Form'
```

Comment:
I could not directly see the code that caused this change.

Reply:
`FormViewSet.parser_classes` and `FormViewSet.renderer_classes` :) the DRF default (in Django settings) allows JSON, FormData and multipart